form.html

```diff
@@ -0,0 +1,220 @@
+      justify-content: center;
+    }
+
+    .flex-column {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    .statement {
+      font-family: SpecialElite;
+      font-size: 1.25rem;
+      line-height: 1.786;
+      word-spacing: 0.1rem;
+      text-align: center;
+      font-size: 0.938rem;
+      color: #dd4d19;
+      margin-top: 1.25rem;
+      text-shadow: 0.25rem 0.25rem 0.25rem #00000040;
+      display: flex;
+      flex-direction: column;
+    }
+
+    #txt-area.box {
+      margin-top: 2rem;
+      width: 13.875rem;
+      height: 7.805rem; /*aspec ratio is 270px x 151.888px */
+      border-radius: 0.313rem;
+      box-shadow: 0.25rem 0.25rem 0.25rem #dd4d19,
+        0.25rem 0.25rem 0.25rem #e27c57;
+      resize: vertical;
+      padding: 2%;
+      font-size: 1.2rem;
+      font-family: SpecialElite;
+      border: 0.2rem dashed #dd4d19;
+      letter-spacing: 0.2rem;
+      word-spacing: 0.4rem;
+      line-height: 1.4rem;
+      display: flex;
+      flex-direction: column;
+      align-self: center;
+    }
+
+    #txt-area.box:hover,
+    #txt-area.box:focus {
+      color: #e27c57;
+      outline: none !important;
+      border-color: #4dbba6;
+      box-shadow: 0 0 10px #4dbba6;
+    }
+
+    ::selection {
+      color: white;
+      background-color: #e27c57;
+    }
+
+    .bttn-3 {
+      box-shadow: 0.4rem 0.25rem 0.25rem #00000040,
+        0.188rem 0.188rem 0.188rem #00000052;
+      background-color: #dd4d19;
+      color: #ffffff;
+      font-family: SpecialElite;
+      font-size: 1.25rem;
+      font-weight: 20;
+      text-shadow: 0.313rem 0.313rem 0.313rem #00000050;
+      width: 10rem;
+      height: 4.063rem;
+      border-color: transparent;
+      border-radius: 0.063rem;
+      position: relative;
+      top: 0.625rem;
+      margin-top: 3rem;
+      align-self: center;
+    }
+
+    #citation {
+      font-family: SpecialElite;
+      font-size: 0.9rem;
+      color: #e27c57;
+      text-align: center;
+      margin-top: 3rem;
+      margin-bottom: 0.5rem;
+    }
+
+    #bell-hooks {
+      font-family: SpecialElite;
+      font-style: bold;
+      font-size: 0.9rem;
+      text-align: center;
+      color: #dd4d19;
+      margin-top: 0rem;
+      margin-bottom: 1rem;
+    }
+
+    @media only screen and (min-width: 414px) {
+      .statement {
+        font-size: 1.5rem;
+        line-height: 2rem;
+        word-spacing: 0.15rem;
+      }
+
+      #txt-area.box {
+        width: 16.875rem;
+        height: 9.493rem;
+      }
+    }
+
+    @media only screen and (min-width: 567px) {
+      .statement {
+        font-size: 1.8rem;
+        line-height: 2.5rem;
+        word-spacing: 0.5rem;
+      }
+
+      #txt-area.box {
+        width: 24.875rem;
+        height: 13.912rem;
+      }
+
+      .bttn-3 {
+        font-size: 1.75rem;
+        width: 13.5rem;
+        height: 5.063rem;
+      }
+    }
+
+    @media only screen and (min-width: 896px) {
```

Comment:
We could declare these media query min widths as variables (see https://developer.mozilla.org/en-US/docs/Web/CSS/--*)